src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -513,8 +557,10 @@ private void RunBackupThread(PeriodicBackup periodicBackup, BackupTask backupTas
                     tcs.SetResult(backupResult);
                 }
             }
-            catch (OperationCanceledException oce)
+            catch (Exception e) when (e.ExtractSingleInnerException() is OperationCanceledException oce)
             {
+                runningBackupStatus.DelayUntil = GetBackupStatus(periodicBackup.BackupStatus.TaskId).DelayUntil;
```

Comment:
`GetBackupStatus` is using the in-memory state. Need to verify that we get the value correctly.

Reply:
Done.